tests/PHPStan/Analyser/data/non-empty-string.php

```diff
@@ -302,19 +302,20 @@ class MoreNonEmptyStringFunctions
 
 	/**
 	 * @param non-empty-string $nonEmpty
+	 * @param '1'|'2'|'5'|'10' $constUnion
```

Comment:
I thought about a test-case mixing `int` and `string` constants in a union.
this would not work with the suggested `TypeUtils::getConstantStrings` method, therefore I went with a strings-only union for now